internal/plugin/plugin.go

```diff
@@ -207,9 +241,10 @@ func createSymlink(root, filePath string) error {
 	return errors.WithStack(os.Symlink(filePath, newname))
 }
 
-func (m *Manager) shouldCreateFile(filePath string) bool {
-	// Only create devboxDir files in add mode.
-	if strings.Contains(filePath, devboxDirName) && !m.addMode {
+func (m *Manager) shouldCreateFile(pkg *lock.Package, filePath string) bool {
+	// Only create files in devboxDir is they are not in the lockfile
```

Comment:
typo: if

Reply:
🤔 for older plugins (i.e. not in the lockfile), this code is re-creating their `devbox.d` files.

- I thought users were encouraged to commit them to source-control and possibly change them.
- so, shouldn't we avoid re-generating these files for existing devbox projects

Reply:
I'll make sure we don't overwrite them, but yeah this will cause a one time re-creation if they previously deleted/moved them. I think that's on ok tradeoff in order to migrate to a better system.

Reply:
Got it

Yeah, I was concerned about the overwriting scenario where users may have made changes.